core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -953,6 +953,15 @@ private SqlLibraryOperators() {
           OperandTypes.or(OperandTypes.STRING, OperandTypes.STRING_STRING),
           SqlFunctionCategory.TIMEDATE);
 
+  @LibraryOperator(libraries = {BIG_QUERY})
+  public static final SqlFunction STRING_FORMAT =
```

Comment:
is there any function in bigquery as STRING_FORMAT because I can't find any

Reply:
As there is no STRING_FORMAT function in BigQuery. This is not valid.
STRING_FORMAT is a keyword.
https://cloud.google.com/bigquery/docs/reference/standard-sql/conversion_functions
https://cloud.google.com/bigquery/docs/reference/standard-sql/conversion_functions#formatting_syntax

Refer this doc and try finding another solution to this.
Discuss with someone that how can we incorporate this type of CAST in our code.